src/core/gui/dialog/LatexSettingsPanel.cpp

```diff
@@ -151,8 +151,7 @@ void LatexSettingsPanel::checkDeps() {
                 if (g_subprocess_wait_check(*proc, nullptr, &err)) {
                     msg = _("Sample LaTeX file generated successfully.");
                 } else {
-                    msg = FS(_F("Error: {1}. Please check the contents of {2}") % err->message %
-                             tmpDir.u8string().c_str());
+                    msg = FS(_F("Error: {1}. Please check the contents of {2}") % err->message % tmpDir.u8string());
```

Comment:
Actually, we should use std::format /{fmt} instead of our own formater. Ironically, we already use a similar format syntax, but our self implemented formater is very inefficient and error-prone. {fmt}/std::format are already compitletime type checked.